spec/models/rook_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+RSpec.describe Rook, type: :model do
+  describe "#valid_move?" do
+    before(:all) do
```

Comment:
Yes, I agree.  I was being a bit timid and wanted to make sure it works.  I will refactor it.